apps/gp2-server/src/routes/public/user.route.ts

```diff
@@ -39,24 +39,28 @@ export const userRouteFactory = (userController: UserController): Router => {
 const mapUserToPublicUser = (
   user: gp2Model.UserResponse,
 ): gp2Model.PublicUserResponse => ({
-  id: user.id,
-  displayName: user.displayName,
+  avatarUrl: user.avatarUrl,
+  biography: user.biography,
+  city: user.city,
   country: user.country,
   degrees: user.degrees,
+  displayName: user.displayName,
   firstName: user.firstName,
+  id: user.id,
   lastName: user.lastName,
-  publishDate: user.createdDate,
-  avatarUrl: user.avatarUrl,
-  biography: user.biography,
-  city: user.city,
+  lastModifiedDate: user.lastModifiedDate,
```

Comment:
Shouldn't it be publishDate?

![Screenshot 2024-06-04 at 07 42 52](https://github.com/yldio/asap-hub/assets/16595804/75d665ca-3181-428a-a79e-e35f29e81578)

Reply:
it is https://github.com/yldio/asap-hub/blob/master/apps/gp2-server/src/data-providers/user.data-provider.ts#L310 - in the case of users the last modified date was already present so I figured this would be easier than copying the publish date again